services/src/main/java/org/exoplatform/addons/gamification/utils/Utils.java

```diff
@@ -176,4 +177,8 @@ public static Long countAnnouncementsByChallenge(Long challengeId) {
       return 0l;
     }
   }
+   public static Long getUserGlobalScore( String earnerId){
+    return StringUtils.isBlank(earnerId) ? null : CommonsUtils.getService(GamificationService.class).computeTotalScore(earnerId);
+   }
```

Comment:
change this please see example : https://github.com/exoplatform/agenda/blob/develop/agenda-services/src/main/java/org/exoplatform/agenda/notification/builder/DatePollNotificationBuilder.java#L136-L141